src/main/java/org/folio/circulation/domain/UserRepository.java

```diff
@@ -91,6 +91,10 @@ public UserRepository(Clients clients) {
       .distinct()
       .collect(Collectors.toList());
 
+    if (usersToFetch.isEmpty()) {
+      return CompletableFuture.completedFuture(HttpResult.succeeded(multipleRequests));
```

Comment:
Similarly, I might use static imports to shorten this code to something like
```suggestion
      return completedFuture(succeeded(multipleRequests));
```